gallery/workbook/Azure Monitor.json

```diff
@@ -108,10 +108,10 @@
 					"name": "Multi Cluster Event Monitoring for Dell APEX Cloud Platform for Microsoft Azure"
 				},
 				{
-					"id": "Workbooks/Azure Stack HCI/PerformanceMetrics",
+					"id": "Workbooks/Azure Stack HCI/PerformanceMetrics/MultiCluster",
```

Comment:
since this template is VERY new probably not a problem, but, this folderpath  is treated as an "id", so someone sharing a link to this template, the previous one won't be found.

so for future reference, renaming/moving folders around in the generally discouraged